plugins/infinispan4/src/main/java/org/radargun/service/InfinispanEmbeddedService.java

```diff
@@ -4,6 +4,7 @@
 import java.lang.management.ManagementFactory;
 import java.util.HashMap;
 import java.util.Map;
+
```

Comment:
-1 ;)

Reply:
I needed to update https://github.com/radargun/radargun/pull/603/files#diff-04dc5d029560773ac79faaf3da0fbb22R800 in order to have https://github.com/radargun/radargun/pull/603/files#diff-a0f6c488d3bebd7797129dd3f9ed36b8R1 working.
It starts reporting those failures and I only fixed otherwise we won't have the build.

Reply:
Aren't the checkstyle failures because of https://github.com/radargun/radargun/pull/603/files#diff-5e19a4b37427e2c0df131bcd48e8f8a3R42 ?